### ISSUE TITLE
[Fedora 32] compilation error

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/arkimet#224]
```
/bin/sh ../libtool  --tag=CXX   --mode=compile g++ -DHAVE_CONFIG_H   -I.. -I.. -L/usr/lib -DCONF_DIR=\"/etc/arkimet\" -DPOSTPROC_DIR=\"/usr/lib64/arkimet\"       -I/usr/include -I/usr/lib64/pkgconfig/../../include  -I/usr/include/mysql -I/usr/include/mysql/..     -Werror  -Wall  -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o matcher/libarkimet_la-area.lo `test -f 'matcher/area.cc' || echo './'`matcher/area.cc
libtool: compile:  g++ -DHAVE_CONFIG_H -I.. -I.. -L/usr/lib -DCONF_DIR=\"/etc/arkimet\" -DPOSTPROC_DIR=\"/usr/lib64/arkimet\" -I/usr/include -I/usr/lib64/pkgconfig/../../include -I/usr/include/mysql -I/usr/include/mysql/.. -Werror -Wall -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c matcher/area.cc  -fPIC -DPIC -o matcher/.libs/libarkimet_la-area.o
matcher/area.cc: In constructor 'arki::matcher::MatchAreaBBox::MatchAreaBBox(const string&, const string&)':
matcher/area.cc:128:29: error: cannot convert 'std::unique_ptr<geos::geom::Geometry>' to 'geos::geom::Geometry*' in assignment
  128 |     this->geom = reader.read(geom);
      |                  ~~~~~~~~~~~^~~~~~
      |                             |
      |                             std::unique_ptr<geos::geom::Geometry>
make[4]: Leaving directory '/root/rpmbuild/BUILD/arkimet-master/arki'
```

full log:
https://travis-ci.org/github/ARPA-SIMC/arkimet/jobs/681492866

internal server available: `makerpm@trentadue`

**Comment by edigiacomo** [comment]
Starting from version 3.8.0 libgeos return [unique_ptr from most methods that produce a new geometry](https://github.com/libgeos/geos/blob/master/NEWS#L54) and `geos::io::WKTReader` is involved (see https://github.com/libgeos/geos/blob/3.7/src/io/WKTReader.cpp#L60-L61 against https://github.com/libgeos/geos/blob/3.8/src/io/WKTReader.cpp#L62-L63).

I can provide a fix based on the geos version.

**Comment by brancomat** [comment]
fixed in version 1.25
